UNITS.md

```diff
@@ -0,0 +1,201 @@
+# All quantities
+ - [AbsorbedDose](#absorbeddose)
+ - [Acceleration](#acceleration)
+ - [AmountOfSubstance](#amountofsubstance)
+ - [Angle](#angle)
+ - [AngularVelocity](#angularvelocity)
+ - [Area](#area)
+ - [Capacitance](#capacitance)
+ - [CatalyticActivity](#catalyticactivity)
+ - [ElectricCharge](#electriccharge)
+ - [ElectricCurrent](#electriccurrent)
+ - [ElectricalConductance](#electricalconductance)
+ - [Energy](#energy)
+ - [Force](#force)
+ - [Frequency](#frequency)
+ - [Illuminance](#illuminance)
+ - [Inductance](#inductance)
+ - [Length](#length)
+ - [LuminousIntensity](#luminousintensity)
+ - [MagneticFlux](#magneticflux)
+ - [MagneticFluxDensity](#magneticfluxdensity)
+ - [Mass](#mass)
+ - [Power](#power)
+ - [Pressure](#pressure)
+ - [Resistance](#resistance)
+ - [Temperature](#temperature)
+ - [Time](#time)
+ - [Velocity](#velocity)
+ - [Voltage](#voltage)
+ - [Volume](#volume)
+
+
+# All Units
+
+## AbsorbedDose
+ - gray
+
+## Acceleration
+ - meterPerSecondSquared
+
+## AmountOfSubstance
+ - mole
+
+## Angle
+ - degree
+ - radian
+
+## AngularVelocity
+ - degreePerSecond
+ - radianPerSecond
+ - rpm
+
+## Area
+ - squareMeter
+
+## Capacitance
+ - farad
+
+## CatalyticActivity
+ - katal
+
+## ElectricCharge
+ - coulomb
+
+## ElectricCurrent
+ - ampere
+ - attoampere
+ - exaampere
+ - femtoampere
+ - gigaampere
+ - kiloampere
+ - megaampere
+ - microampere
+ - milliampere
+ - nanoampere
+ - petaampere
+ - picoampere
+ - teraampere
+ - yoctoampere
+ - yottaampere
+ - zeptoampere
+ - zettaampere
+
+## ElectricalConductance
+ - siemens
+
+## Energy
+ - joule
+
+## Force
+ - newton
+
+## Frequency
+ - becquerel
+ - herz
```

Comment:
I believe it is spelled "Her**t**z": https://en.wikipedia.org/wiki/Hertz

Reply:
thanks